src/main/java/org/octopusden/octopus/license/management/plugins/gradle/dto/ArtifactGAV.java

```diff
@@ -37,16 +40,20 @@ public String getExtension() {
         return extension;
     }
 
+    public List<ExcludeRule> getExcludeRules() {
+        return excludeRules;
+    }
+
     @Override
     public boolean equals(Object o) {
         if (this == o) return true;
         if (!(o instanceof ArtifactGAV)) return false;
         ArtifactGAV that = (ArtifactGAV) o;
-        return Objects.equals(group, that.group) && Objects.equals(artifact, that.artifact) && Objects.equals(version, that.version) && Objects.equals(classifier, that.classifier) && Objects.equals(extension, that.extension);
+        return Objects.equals(group, that.group) && Objects.equals(artifact, that.artifact) && Objects.equals(version, that.version) && Objects.equals(classifier, that.classifier) && Objects.equals(extension, that.extension) && excludeRules != null ? excludeRules.equals(that.excludeRules) : that.excludeRules == null;
```

Comment:
without paranthesis it will be wrong order of operators
&& (excludeRules != null ? excludeRules.equals(that.excludeRules) : that.excludeRules == null);